public/dorequest.php

```diff
@@ -114,6 +120,52 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
     return DoRequestError("You do not have permission to do that.", 403, 'access_denied');
 }
 
+/*
+ * It is possible for some calls to be made on behalf of another user.
+ * This is only currently supported for "Standalone" integrations.
+ * NOTE: "awardachievements" is not included here because it accepts an array of
+ * achievement ID values. It doesn't use the generic `delegateUserAction()` function.
+ */
+$allowsGenericDelegation = [
+    "awardachievement",
+    "ping",
+    "startsession",
+];
+if (
+    in_array($requestType, $allowsGenericDelegation)
+    && $delegateTo !== null
+    && ($gameID || $achievementID)
+) {
+    if (request()->method() !== 'POST') {
+        return DoRequestError('Access denied.', 403, 'access_denied');
```

Comment:
I believe this should be 405 (Method Not Allowed). The code string can probably remain `access denied`.

Reply:
Fixed in latest.